README.md

```diff
@@ -940,6 +940,11 @@ GET /snapshot/detail/legacy?stacks={stacks}
 ```
 > Returns a list of all applications that have been deployed on legacy stacks. Replace `{stacks}` request parameter-value above with a comma-separated list of legacy stacks like `windows2012R2,cflinuxfs2`.
 
+```
+GET /snapshot/detail/legacy?service-offerings={service-offerings}
+```
+> Returns a list of all service instances matching a product slug name as defined in the comma-separated list. Replace `{service-offerings}` request parameter-value above with a comma-separated list of legacy service-offerings like `p.config-server,p-mysql`.
```

Comment:
Please adjust the end of text to read: "list of legacy service offerings like `p-config-server`, `p-service-registry`, `p-mysql`.